assets/js/modules/thank-with-google/components/common/PositionRadio.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import {
+	MODULES_THANK_WITH_GOOGLE,
+	BUTTON_PLACEMENT_STATIC_ABOVE_CONTENT,
+	BUTTON_PLACEMENT_STATIC_AUTO,
+	BUTTON_PLACEMENT_STATIC_BELOW_1ST_P,
+	BUTTON_PLACEMENT_STATIC_BELOW_CONTENT,
+} from '../../datastore/constants';
+import Radio from '../../../../components/Radio';
+const { useSelect, useDispatch } = Data;
+
+export default function PositionRadio() {
+	const { setButtonPlacement } = useDispatch( MODULES_THANK_WITH_GOOGLE );
+
+	const buttonPlacement = useSelect( ( select ) =>
+		select( MODULES_THANK_WITH_GOOGLE ).getButtonPlacement()
+	);
+
+	const onChange = useCallback(
+		( { target } = {} ) => {
+			const { value: placement } = target || {};
+			setButtonPlacement( placement );
+		},
+		[ setButtonPlacement ]
+	);
+
+	return (
+		<div className="googlesitekit-twg-setting-field googlesitekit-twg-position-radio">
+			<h4>{ __( 'Position', 'google-site-kit' ) }</h4>
+			<div className="googlesitekit-twg-position-radio__option">
+				<Radio
+					id={ `button-placement-${ BUTTON_PLACEMENT_STATIC_AUTO }` }
+					name="button-placement"
+					value={ BUTTON_PLACEMENT_STATIC_AUTO }
+					checked={ buttonPlacement === BUTTON_PLACEMENT_STATIC_AUTO }
+					onChange={ onChange }
+				>
+					{ __( 'Auto', 'google-site-kit' ) }
+				</Radio>
+				<p>
+					{ __(
+						'The prompt is automatically placed where it may perform best',
+						'google-site-kit'
+					) }
+				</p>
+			</div>
+			<div className="googlesitekit-twg-position-radio__option">
+				<Radio
+					id="button-placement-manual"
+					name="button-placement"
+					value=""
+					readonly
+				>
+					{ __( 'Manual', 'google-site-kit' ) }
+				</Radio>
```

Comment:
I just checked and can see this is indeed what was [laid out in the Figma design](https://www.figma.com/file/t5zDpQ28Lo7ZyRcOpGdqgu/TwG?node-id=1205%3A7472), but we should change it 🙂 

Unfortunately this radio button doesn't _look_ or _behave_ like a read-only button for both sighted keyboard users and sighted mouse users—it looks and behaves just like the "Auto" button but never changes. At best this is buggy and awkward—I haven't tried using it with a screenreader—in that case the `readonly` attribute might prevent SOME confusion, but given it can _never_ be interactive and is functioning more like a heading/separator than a button, we shouldn't use a `Radio` button component here at all. Otherwise it behaves like this which is quite confusing:

![CleanShot 2022-07-27 at 20 31 45](https://user-images.githubusercontent.com/90871/181357147-64b88b46-111f-494f-bf0d-3a511d0b7fb5.gif)

So let's deviate from the design and make this not a radio button so it doesn't appear in the keyboard order, doesn't _look_ like it can be clicked, etc.

Reply:
I have updated it to be disabled which makes it inactive. Does it look good to you now?

Reply:
It's _better_ that way, especially for keyboard users, but visually it's still "off" to me, because there's a cursor change when hovering over the radio button (implying it can be clicked) and it visually looks like it's a button.

If a button is disabled 100% of the time, it shouldn't be a button.

But to prevent this issue from blocking other issues I've filed a follow-up issue to fix this: #5623. I think it's an issue with the source design and the `disabled` hack is certainly good enough for now. 👍🏻